src/main/java/in/mohamedhalith/controller/AuthController.java

```diff
@@ -22,28 +23,31 @@
 
 @RestController
 public class AuthController {
+
 	@Autowired
 	EmployeeService employeeService;
 	@Autowired
 	LoginValidator loginValidator;
+
 	@PostMapping("LoginServlet")
-	public AuthDTO login(@RequestBody LoginDTO user,HttpServletRequest request) throws ServiceException, ValidationException {
```

Comment:
why create separate dto for login purpose ? use existing model class(user )

Reply:
> why create separate dto for login purpose ? use existing model class(user )

Model class employee contains many unrelated fields and the naming may confuse. hence i created a seperate dto with necessary fields